pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// ----------------------------
+// #SETTINGS
+// ----------------------------
+@mixin alert-base($alert-color) {
+    border-top: rem(4) solid $alert-color;
+    .alert-icon {
+        color: palette(grayscale, white);
+        background-color: $alert-color;
+    }
+}
+
+// ----------------------------
+// #GENERAL
+// ----------------------------
+.alert {
+    border: rem(1) solid palette(grayscale-cool, light);
+    border-radius: rem(3);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    overflow: auto;
+}
+
+.alert-icon {
+    // create a circle around the icon
+    border-radius: 50%;
+
+    // create room around the icon for the circle
+    padding: rem(7);
+
+    @include susy-media($bp-screen-md) {
+        float: left;
+    }
+
+}
+
+.alert-message {
+    padding: spacing-vertical(small);
+    padding-left: 0;
+    padding-right: 0;
+
+    @include susy-media($bp-screen-md) {
+        float: left;
+        width: 70%;
+        padding: spacing-vertical(small);
+        padding-top: 0;
+        padding-bottom: 0;
+
+        // shift the message down to be in line with the icon
+        margin-top: rem(3);
```

Comment:
This seems like a magic number (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#imperfect-and-magic-numbers). Any way you can use one of our values from the ``spacing-vertical`` Sass map instead?